.github/workflows/pkg-pr-new.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish Any Commit
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      # Label used to access the service container
+      postgres:
```

Comment:
This is only needed for testing, so it can be removed if you end up not running tests below